apps/web/__tests__/support/pageObjects/SiteSettingsObject.ts

```diff
@@ -0,0 +1,122 @@
+import { PageObject } from "./PageObject";
+
+export class SiteSettingsObject extends PageObject {
+  get settingsDrawer() {
+    return cy.getByTestId('site-settings-drawer');
+  }
+  get fontSection() {
+    return cy.getByTestId('font-section');
+  }
+
+  get colorSection() {
+    return cy.getByTestId('color-section');
+  }
+
+  get blockSpacingSection() {
+    return cy.getByTestId('block-spacing-section');
+  }
+
+  get fontInput() {
+    return cy.getByTestId('font-select');
+  }
+
+  get body() {
+    return cy.getByTestId('body');
+  }
+
+  get navbar() {
+    return cy.getByTestId('navbar-top');
+  }
+
+  get block() {
+    return cy.getByTestId('block-wrapper');
+  }
+
+  get saveButton() {
+    return cy.getByTestId('edit-save-button');
+  }
+
+  get primaryColorInput() {
+    return cy.getByTestId('primary-color-select');
+  }
+
+  get secondaryColorInput() {
+    return cy.getByTestId('secondary-color-select');
+  }
+
+  get blockSpacingButton() {
+    return cy.getByTestId(`block-spacing-btn`);
+  }
+
+  checkDrawerVisible() {
+    this.settingsDrawer.should('be.visible');
+    return this;
+  }
+
+  checkDrawerNotVisible() {
+    this.settingsDrawer.should('not.exist');
+    return this;
+  }
+
+  checkSaveBtnDisabled() {
```

Comment:
```suggestion
  checkSaveButtonDisabled() {
```
We're not really saving anything by typing three characters less.